gk/main.c

```diff
@@ -634,6 +711,34 @@ gk_proc(void *arg)
 			process_gk_cmd(gk_cmds[i], instance);
 			mb_free_entry(&instance->mb, gk_cmds[i]);
         	}
+
+		if (ipv6_if_configured(&gk_conf->net->front) && acl.num > 0) {
+			ret = rte_acl_classify(
```

Comment:
Ok, `rte_acl_classify()` uses the algorithm set in the ACL context, and I found your XXX comment pushing the use of `rte_acl_set_ctx_classify()` to later. Is it hard to find the supported hardware and set the right algorithm?